pkg/pillar/agentlog/agentlog.go

```diff
@@ -317,10 +368,38 @@ func roundToMb(b uint64) uint64 {
 	return mb
 }
 
+// Send application stdout/stderr to this panic file
+func spoofStdFDs(panicFileName string) *os.File {
+	filename := fmt.Sprintf("%s/%s", types.PersistPanicDir, panicFileName)
+	panicOut, _ := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_SYNC|os.O_APPEND, 0755)
+	fd2, err := syscall.Dup(int(os.Stdout.Fd()))
+	if err != nil {
+		log.Fatalf("Error duplicating Stdout: %s", err)
+	}
+	originalStdout, err := os.OpenFile(fmt.Sprintf("/dev/fd/%d", fd2),
+		os.O_WRONLY|os.O_CREATE|os.O_SYNC, 0755)
+	if err != nil {
+		log.Fatalf("Error opening duplicate stdout with fd: %v", fd2)
+	}
+	// replace stdout
+	err = syscall.Dup2(int(panicOut.Fd()), 1)
+	if err != nil {
```

Comment:
Why are we replacing stdout @gkodali-zededa -- panic goes only to stderr

Reply:
Idea was to capture anything that did not go into the logging system here.